website/docs/r/alert_policy.html.markdown

```diff
@@ -33,8 +33,8 @@ The following arguments are supported:
 In addition to all arguments above, the following attributes are exported:
 
   * `id` - The ID of the policy.
-  * `created_at` - The time the policy was created.
-  * `updated_at` - The time the policy was last updated.
+  * `created_at` - **DEPRECATED** The time the policy was created.
```

Comment:
Add colons after DEPRECATED for consistency with @sanderblue 's PR?

https://github.com/terraform-providers/terraform-provider-newrelic/pull/505

Reply:
Yep, I think we're moving forward **DEPRECATED:**

Reply:
Can do.

Reply:
Done.